src/repos/ArticleRepo.ts

```diff
@@ -103,6 +116,12 @@ const incrementShoutouts = async (id: string): Promise<Article> => {
   return article.save();
 };
 
+const isCovidArticle = (title: string): boolean => {
+  const filter = new Filter();
+  filter.addWords("covid", "covid-19", "coronavirus", "test", "mask", "masks", "pandemic", "tests", "omicron","endemic")
```

Comment:
Does this filter automatically take care of lowercasing text? So we don't miss 'COVID' because the word we added is 'covid'.

Also, as discussed before, adding 'test' and 'tests' may be a little too restrictive as it could block articles about actual exams.

Reply:
We only have access to the article bodies in the microservice. I think it could be possible to do the check there and still not damage the article data.